arex-instrumentation-api/src/main/java/io/arex/inst/runtime/util/CaseManager.java

```diff
@@ -45,4 +46,15 @@ public static boolean isInvalidCase(String traceId) {
         }
         return context.isInvalidCase();
     }
+
+    public static Object getForceRecordAttachment(String recordId) {
```

Comment:
This method is more appropriately placed in a `ContextManager`

Reply:
`public Object getAttachment(String traceId, String key)`

Method signatures can be adjusted to be more generic.

- replay scene: traceId is replayId
- record scene: traceId is recordId